File: DDA Service/Service/FoodNutrient.cs
using System.Runtime.Serialization;

namespace Service
{
    [DataContract]
    public class FoodNutrient
    {
        [DataMember]
        string name;    //이름
        [DataMember]
        byte[] img;     //사진
        [DataMember]
        int kcal;       //열량
        [DataMember]
        double cho;     //탄수화물
        [DataMember]
        double fat;     //지방
        [DataMember]
        double protein; //단백질
        [DataMember]
        double na;      //나트륨
        
        public FoodNutrient() { }

        public FoodNutrient(string name, int kcal, double cho, double fat, double protein, double na)
        {
            this.name = name;
            this.kcal = kcal;
            this.cho = cho;
            this.fat = fat;
            this.protein = protein;
            this.na = na;           
        }

        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public byte[] IMG
        {
            get { return img; }
            set { img = value; }
        }

        public int Kcal
        {
            get { return kcal; }
            set { kcal = value; }
        }

        public double Cho
        {
            get { return cho; }
            set { cho = value; }
        }

        public double Fat
        {
            get { return fat; }
            set { fat = value; }
        }

        public double Protein
        {
            get { return protein; }
            set { protein = value; }
        }

        public double Na
        {
            get { return na; }
            set { na = value; }
        }
               
    }
}

File: DDAAplication/app/src/main/java/com/example/dktkg/ddatest/FoodNutrient.java
package com.example.dktkg.ddatest;

import java.io.Serializable;

/**
 * Created by dktkg on 2017-05-22.
 */

public class FoodNutrient implements Serializable {
    private String name;    //이름
    private byte[] img;     //사진
    private int kcal;       //열량
    private double cho;     //탄수화물
    private double fat;     //지방
    private double protein; //단백질
    private double na;      //나트륨



    public FoodNutrient(String name, byte[] img, int kcal, double cho, double fat, double protein, double na) {
        this.name = name;
        this.img = img;
        this.kcal = kcal;
        this.cho = cho;
        this.fat = fat;
        this.protein = protein;
        this.na = na;

    }

    public FoodNutrient(int kcal, double cho, double fat, double protein, double na) {

        this.kcal = kcal;
        this.cho = cho;
        this.fat = fat;
        this.protein = protein;
        this.na = na;

    }

    public String getName() {
        return name;
    }

    public byte[] getImg() {
        return img;
    }

    public int getKcal() {
        return kcal;
    }

    public double getCho() {
        return cho;
    }

    public double getFat() {
        return fat;
    }

    public double getProtein() {
        return protein;
    }

    public double getNa() {
        return na;
    }

    public void setName(String name) {
        this.name = name;
    }

}
